torchaudio/models/wav2vec2/components.py

```diff
@@ -273,15 +275,22 @@ def forward(
         self,
         x: Tensor,
         attention_mask: Optional[Tensor] = None,
-    ) -> Tensor:
+        position_bias: Optional[Tensor] = None,
+        key_padding_mask: Optional[Tensor] = None,
+    ) -> Tuple[Tensor, Optional[Tensor]]:
         """
         Args:
             x (Tensor): shape: ``[batch_size, sequence_length, embed_dim]``.
-            attention_mask (Tensor or None, optional):
+            attention_mask (Tensor or ``None``, optional):
                 shape: ``[batch_size, 1, sequence_length, sequence_length]``
-
+            position_bias (Tensor or ``None``): Has to be ``None``, necessary for compatibility
+                with :py:class:`WavLMSelfAttention`. (Default: ``None``)
+            key_padding_mask (Tensor or ``None``): Has to be ``None``, necessary for compatibility
+                with :py:class:`WavLMSelfAttention`. (Default: ``None``)
         Returns:
-            Tensor: The resulting tensor. shape: ``[batch, sequence_length, embed_dim]``
+            (Tensor, ``None``): The resulting attention output and ``None`` (necessary for compatibility
+                with :py:class:`WavLMSelAttention`).
+                Attention output shape: ``[batch, sequence_length, embed_dim]``.
```

Comment:
> necessary for compatibility

I understand why this change is necessary but this feels like code smell... but I do not know if there is a better way either... It's okay as this is not a public API 🧐🧐🧐